.readthedocs.yaml

```diff
@@ -0,0 +1,24 @@
+# .readthedocs.yaml
```

Comment:
This file is unrelated to this PR.

Reply:
It is. I just thought I'll use the occasion to add it.
I'll move it to a separate PR.